index.js

```diff
@@ -1,55 +1,408 @@
+// removes all of the element's descendants and replaces them with nodes constructed by parsing the HTML given in the string htmlString.
+
+// *createElement*
+// A string that specifies the type of element to be created. The nodeName of the created element is initialized with the value of tagName. 
+// Don't use qualified names (like "html:a") with this method. When called on an HTML document, createElement() converts tagName to lower
+// case before creating the element. In Firefox, Opera, and Chrome, createElement(null) works like createElement("null")
+
+// *append*
+// The Element.append() method inserts a set of Node objects or string objects after the last child of the Element. String objects are 
+// inserted as equivalent Text nodes.
+
+// *.className*
+// The className property of the Element interface gets and sets the value of the class attribute of the specified element.
+
+// *addEventListener*
+// The addEventListener() method of the EventTarget interface sets up a function that will be called whenever the specified event is delivered
+// to the target. Common targets are Element, or its children, Document, and Window, but the target may be any object that supports events 
+// (such as XMLHttpRequest). The method addEventListener() works by adding a function, or an object that implements EventListener, to the list 
+// of event listeners for the specified event type on the EventTarget on which it's called. If the function or object is already in the list of 
+// event listeners for this target, the function or object is not added a second time. 
+
+// *indexOf*
+// The indexOf() method returns the first index at which a given element can be found in the array, or -1 if it is not present.
+
+// *preventDefault*
+// The preventDefault() method of the Event interface tells the user agent that if the event does not get explicitly handled, 
+// its default action should not be taken as it normally would be.The event continues to propagate as usual, unless one of its
+// event listeners calls stopPropagation() or stopImmediatePropagation(), either of which terminates propagation at once.
+
+// *setAttribute*
+// Sets the value of an attribute on the specified element. If the attribute already exists, the value is updated; otherwise a 
+// new attribute is added with the specified name and value.
+
+// *localeCompare*
+// The localeCompare() method returns a number indicating whether a reference string comes before, or after, or is the same as
+//  the given string in sort order.
+
+// STEP #1
+// declare a const containing all the items we wish to show on the screen above the basket.
+// these items, along with the "+ or -" button will be displayed in the basket when "add to cart" under the img is clicked
+// each item should contain a name, id, price, and type. These should all be placed in an array so that individual items can
+// be called upon.
+
```

Comment:
Very well noted lol

Reply:
I know I was a bit excessive with my notes lol, but i literally need to annotate everything so when i need to refer back, i have a basic guide on the previous processes. I find it helps speed things up for me when reviewing and editing